src/components/button/button.js

```diff
@@ -1,10 +1,30 @@
 import React from "react";
 import PropTypes from "prop-types";
 
-export const Button = ({ children }) => {
-  return <button className="button is-danger is-outlined">{children}</button>;
+export const Button = ({ children, onClick, type, inverted }) => {
+  const additionalClasses = [];
+
+  if (type) {
+    additionalClasses.push(`is-${type}`);
+  }
+
+  if (inverted) {
+    additionalClasses.push("is-outlined");
+  }
+
+  return (
+    <button
+      onClick={onClick}
+      className={"button is-primary " + additionalClasses.join(" ")}
```

Comment:
We should not hardcode `is-primary` here. It's true it's gonna be overriden by the other types, but in this case we won't use the styles from it.
```suggestion
      className={"button " + additionalClasses.join(" ")}
```

Reply:
Also, for dynamic classes we could use [classNames](https://github.com/JedWatson/classnames#usage), you have an example on this [wip branch](https://github.com/code4romania/taskforce-fe-components/blob/dan/%234_add-list-components/src/components/ListItem/ListItem.js#L10)

Reply:
That `is-primary` there was forgotten, my bad.

As for the classNames, I simplified the code for that. Please advise if you still consider adding an extra package for this looks necessary.